devicon.json

```diff
@@ -6654,6 +6654,27 @@
         "color": "#dd4814",
```

Comment:
What's the reason this (`#dd4814`) color got chosen as the main color, as opposed to the other colors from the official logo (`#962444`, `#FBBD16`, `#4A2469`, `452E7F`)? Can't find the color anywhere in their logo.

Official logo: https://www.uml.org/
```svg
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="121px" height="88px" viewBox="0 0 396 288" enable-background="new 0 0 396 288" xml:space="preserve">
<polygon fill="#FBBD16" points="208.096,234.496 208.096,158.167 222.729,240.603 246.553,252.477 261.867,183.949 261.867,258.923 
	288.753,272.151 288.753,145.616 247.914,126.278 234.982,186.663 221.937,113.84 181.323,94.391 181.323,221.04 "></polygon>
<polygon fill="#452E7F" points="292.496,145.583 335.152,123.418 335.152,215.238 394.142,184.709 394.369,219.31 292.611,272.796 
	"></polygon>
<path fill="#962444" d="M275.298,12.158l-30.163,26.669c0,0,58.851,30.25,66,33.825c7.15,3.574,17.325,10.449,6.325,20.625
	c-11,10.175-23.1,8.8-31.35,4.675S216.81,63.027,216.81,63.027l-31.074,26.399c0,0,59.949,30.525,69.85,35.75
	c9.9,5.226,46.75,18.15,81.675-9.075c34.925-27.225,29.976-37.125,29.976-39.05s-0.825-12.1-6.875-18.425
	c-6.051-6.325-23.65-15.4-31.075-19.25S275.298,12.158,275.298,12.158z"></path>
<path fill="#4A2469" d="M50.154,100.832v18.721c0,2.123-0.057,3.614-0.17,4.473c-0.113,0.859-0.448,1.741-1.006,2.646
	c-0.557,0.906-1.29,1.594-2.202,2.06c-0.911,0.468-1.985,0.7-3.222,0.7c-1.369,0-2.577-0.275-3.626-0.829
	c-1.048-0.555-1.831-1.275-2.352-2.163c-0.519-0.889-0.825-1.825-0.92-2.812s-0.141-3.061-0.141-6.221v-16.575h5.968v21.002
	c0,1.223,0.055,2.005,0.162,2.344c0.107,0.341,0.326,0.511,0.653,0.511c0.374,0,0.615-0.188,0.725-0.563
	c0.107-0.375,0.161-1.26,0.161-2.655v-20.639H50.154z"></path>
<path fill="#4A2469" d="M69.866,100.832v28.007h-5.237l-3.104-12.732v12.732h-4.98v-28.007h5.001l3.34,12.611v-12.611H69.866z"></path>
<path fill="#4A2469" d="M82.308,100.832v28.007h-5.969v-28.007H82.308z"></path>
<path fill="#4A2469" d="M86.875,100.832h12.017v5.612h-4.929v5.297h4.39v5.337h-4.39v11.761h-7.088V100.832z"></path>
<path fill="#4A2469" d="M110.365,100.832v28.007h-5.969v-28.007H110.365z"></path>
<path fill="#4A2469" d="M116.83,100.832h9.942v5.612h-3.974v5.297h3.717v5.337h-3.717v6.148h4.368v5.612H116.83V100.832z"></path>
<path fill="#4A2469" d="M132.965,100.832h4.47c2.886,0,4.836,0.161,5.854,0.483c1.018,0.324,1.791,0.854,2.32,1.592
	s0.86,1.56,0.994,2.466c0.132,0.905,0.198,2.684,0.198,5.336v9.809c0,2.516-0.097,4.195-0.291,5.043
	c-0.194,0.849-0.533,1.512-1.015,1.99c-0.482,0.479-1.079,0.813-1.788,1.004c-0.709,0.189-1.778,0.284-3.207,0.284h-7.535V100.832z
	 M138.935,105.614v18.442c0.864,0,1.397-0.211,1.598-0.633c0.199-0.421,0.299-1.566,0.299-3.437v-10.892
	c0-1.271-0.033-2.084-0.1-2.441s-0.219-0.621-0.456-0.789C140.038,105.699,139.59,105.614,138.935,105.614z"></path>
<path fill="#4A2469" d="M20.979,146.977v28.009h-5.217l-0.009-18.909l-2.067,18.909H9.969l-2.186-18.494l-0.008,18.494H2.557
	v-28.009h7.725c0.228,1.688,0.464,3.676,0.709,5.959l0.847,7.119l1.359-13.078H20.979z"></path>
<path fill="#4A2469" d="M41.391,163.386c0,2.813-0.054,4.805-0.163,5.976c-0.108,1.169-0.449,2.239-1.021,3.207
	c-0.571,0.969-1.345,1.712-2.318,2.231c-0.975,0.519-2.109,0.778-3.405,0.778c-1.229,0-2.333-0.245-3.312-0.736
	c-0.979-0.489-1.766-1.223-2.362-2.204c-0.595-0.98-0.95-2.046-1.063-3.199c-0.113-1.154-0.17-3.171-0.17-6.053v-4.791
	c0-2.813,0.054-4.804,0.162-5.974c0.11-1.171,0.449-2.239,1.022-3.209c0.572-0.968,1.346-1.712,2.318-2.23
	c0.975-0.52,2.109-0.778,3.404-0.778c1.23,0,2.335,0.246,3.313,0.735c0.978,0.49,1.765,1.226,2.361,2.205
	c0.596,0.979,0.95,2.046,1.063,3.199c0.113,1.152,0.17,3.17,0.17,6.052V163.386z M35.422,154.189c0-1.302-0.06-2.135-0.178-2.497
	c-0.119-0.363-0.363-0.545-0.732-0.545c-0.313,0-0.552,0.147-0.719,0.44c-0.166,0.294-0.249,1.161-0.249,2.602v13.066
	c0,1.625,0.055,2.627,0.163,3.007c0.109,0.382,0.363,0.572,0.762,0.572c0.408,0,0.669-0.221,0.783-0.658
	c0.112-0.437,0.17-1.479,0.17-3.128V154.189z"></path>
<path fill="#4A2469" d="M47.959,146.977h4.47c2.886,0,4.837,0.162,5.854,0.484c1.018,0.324,1.791,0.854,2.32,1.593
	c0.529,0.737,0.86,1.559,0.993,2.464c0.134,0.906,0.199,2.685,0.199,5.337v9.809c0,2.516-0.097,4.196-0.291,5.044
	c-0.193,0.848-0.532,1.509-1.015,1.988c-0.482,0.479-1.078,0.813-1.788,1.003c-0.709,0.19-1.778,0.287-3.207,0.287h-7.535V146.977z
	 M53.929,151.76v18.442c0.865,0,1.397-0.211,1.597-0.633c0.2-0.42,0.3-1.566,0.3-3.438V155.24c0-1.27-0.033-2.083-0.1-2.441
	s-0.218-0.62-0.457-0.788C55.031,151.844,54.585,151.76,53.929,151.76z"></path>
<path fill="#4A2469" d="M68.369,146.977h12.072v5.613h-4.824v5.297h4.512v5.337h-4.512v6.147h5.303v5.614H68.369V146.977z"></path>
<path fill="#4A2469" d="M92.532,146.977v22.395h4.715v5.614H84.795v-28.009H92.532z"></path>
<path fill="#4A2469" d="M106.169,146.977v28.009h-5.97v-28.009H106.169z"></path>
<path fill="#4A2469" d="M126.245,146.977v28.009h-5.238l-3.103-12.733v12.733h-4.981v-28.009h5.001l3.341,12.612v-12.612H126.245z"></path>
<path fill="#4A2469" d="M146.705,157.294h-5.969v-2.538c0-1.6-0.058-2.602-0.17-3.005c-0.113-0.402-0.381-0.604-0.804-0.604
	c-0.366,0-0.615,0.173-0.747,0.519s-0.197,1.233-0.197,2.662v13.43c0,1.256,0.065,2.083,0.196,2.479
	c0.132,0.397,0.395,0.599,0.789,0.599c0.431,0,0.724-0.226,0.877-0.675c0.155-0.45,0.232-1.325,0.232-2.629v-3.319h-1.186v-4.25
	h6.978v15.023h-3.765l-0.555-2.007c-0.406,0.865-0.917,1.515-1.535,1.947c-0.617,0.435-1.347,0.652-2.186,0.652
	c-1,0-1.938-0.296-2.81-0.893c-0.872-0.593-1.535-1.326-1.987-2.203c-0.453-0.878-0.735-1.796-0.85-2.758
	c-0.112-0.963-0.17-2.408-0.17-4.333v-8.319c0-2.673,0.118-4.615,0.354-5.826s0.914-2.321,2.034-3.329
	c1.119-1.009,2.566-1.514,4.342-1.514c1.748,0,3.196,0.438,4.349,1.313c1.152,0.878,1.903,1.918,2.252,3.121
	c0.351,1.206,0.525,2.954,0.525,5.247V157.294z"></path>
<path fill="#4A2469" d="M11.112,192.022v22.395h4.833v5.613H3.177v-28.008H11.112z"></path>
<path fill="#4A2469" d="M31.58,191.705l3.424,28.009h-6.104l-0.318-5.04h-2.128l-0.349,5.04h-6.183l3.048-28.009H31.58z
	 M28.429,209.713c-0.297-3.168-0.6-7.089-0.907-11.767c-0.593,5.369-0.968,9.29-1.127,11.767H28.429z"></path>
<path fill="#4A2469" d="M53.164,192.022v28.008h-5.983l-3.545-12.734v12.734h-5.69v-28.008h5.712l3.817,12.61v-12.61H53.164z"></path>
<path fill="#4A2469" d="M73.392,202.34h-6.785v-2.54c0-1.599-0.064-2.599-0.193-3.005c-0.128-0.402-0.434-0.603-0.913-0.603
	c-0.417,0-0.699,0.173-0.85,0.519c-0.149,0.347-0.225,1.233-0.225,2.661v13.431c0,1.255,0.075,2.083,0.225,2.479
	c0.147,0.397,0.446,0.599,0.895,0.599c0.49,0,0.823-0.226,1-0.675c0.175-0.45,0.264-1.324,0.264-2.63v-3.317H65.46v-4.249h7.932
	v15.021h-4.279l-0.631-2.007c-0.461,0.865-1.043,1.515-1.744,1.947c-0.704,0.435-1.532,0.652-2.487,0.652
	c-1.136,0-2.199-0.296-3.192-0.893c-0.992-0.593-1.745-1.326-2.26-2.203s-0.837-1.796-0.965-2.757
	c-0.13-0.964-0.193-2.41-0.193-4.334v-8.318c0-2.674,0.133-4.616,0.401-5.827c0.269-1.21,1.04-2.32,2.313-3.33
	c1.272-1.009,2.917-1.512,4.937-1.512c1.986,0,3.636,0.437,4.945,1.314c1.311,0.874,2.163,1.914,2.561,3.12
	c0.397,1.203,0.596,2.952,0.596,5.246V202.34z"></path>
<path fill="#4A2469" d="M92.757,192.022v18.721c0,2.122-0.063,3.614-0.186,4.472c-0.124,0.859-0.49,1.741-1.099,2.647
	c-0.609,0.905-1.411,1.593-2.405,2.059c-0.997,0.468-2.17,0.702-3.521,0.702c-1.496,0-2.817-0.276-3.962-0.83
	c-1.146-0.553-2.002-1.275-2.568-2.164c-0.569-0.888-0.904-1.823-1.008-2.812c-0.103-0.986-0.153-3.06-0.153-6.22v-16.575h6.522
	v21.001c0,1.223,0.058,2.006,0.178,2.344c0.116,0.342,0.354,0.513,0.713,0.513c0.409,0,0.673-0.188,0.789-0.564
	c0.117-0.372,0.178-1.26,0.178-2.655v-20.638H92.757z"></path>
<path fill="#4A2469" d="M108.235,192.022l3.424,28.008h-6.104l-0.319-5.04h-2.128l-0.349,5.04h-6.182l3.048-28.008H108.235z
	 M105.085,210.028c-0.298-3.167-0.6-7.089-0.908-11.765c-0.593,5.368-0.969,9.29-1.126,11.765H105.085z"></path>
<path fill="#4A2469" d="M130.512,202.34h-6.65v-2.54c0-1.599-0.063-2.599-0.188-3.005c-0.126-0.402-0.423-0.603-0.896-0.603
	c-0.407,0-0.686,0.173-0.832,0.519c-0.146,0.347-0.22,1.233-0.22,2.661v13.431c0,1.255,0.073,2.083,0.219,2.479
	c0.146,0.397,0.438,0.599,0.878,0.599c0.48,0,0.807-0.226,0.979-0.675c0.173-0.45,0.259-1.324,0.259-2.63v-3.317h-1.321v-4.249
	h7.773v15.021h-4.194l-0.617-2.007c-0.452,0.865-1.022,1.515-1.711,1.947c-0.688,0.435-1.5,0.652-2.435,0.652
	c-1.114,0-2.158-0.296-3.129-0.893c-0.974-0.593-1.712-1.326-2.216-2.203c-0.505-0.877-0.82-1.796-0.946-2.757
	c-0.125-0.964-0.188-2.41-0.188-4.334v-8.318c0-2.674,0.132-4.616,0.396-5.827c0.262-1.21,1.017-2.32,2.264-3.33
	c1.247-1.009,2.86-1.512,4.838-1.512c1.947,0,3.562,0.437,4.846,1.314c1.284,0.874,2.12,1.914,2.51,3.12
	c0.39,1.203,0.584,2.952,0.584,5.246V202.34z"></path>
<path fill="#4A2469" d="M134.736,192.022h12.071v5.612h-4.823v5.299h4.512v5.336h-4.512v6.147h5.303v5.613h-12.551V192.022z"></path>
<g>
	<path fill="#6D6E71" d="M393.104,167.064c0,4.471-3.509,7.979-8.076,7.979c-4.519,0-8.123-3.509-8.123-7.979
		c0-4.375,3.604-7.884,8.123-7.884C389.596,159.181,393.104,162.689,393.104,167.064z M378.924,167.064
		c0,3.509,2.596,6.298,6.153,6.298c3.461,0,6.009-2.789,6.009-6.25c0-3.51-2.548-6.346-6.058-6.346
		C381.52,160.767,378.924,163.603,378.924,167.064z M383.779,171.198h-1.827v-7.883c0.722-0.146,1.731-0.241,3.029-0.241
		c1.489,0,2.163,0.241,2.739,0.577c0.433,0.336,0.769,0.961,0.769,1.73c0,0.864-0.672,1.538-1.634,1.827v0.096
		c0.77,0.289,1.202,0.864,1.442,1.923c0.24,1.202,0.385,1.683,0.576,1.971h-1.97c-0.241-0.288-0.385-1.01-0.626-1.923
		c-0.144-0.865-0.625-1.25-1.635-1.25h-0.864V171.198z M383.827,166.728h0.865c1.01,0,1.827-0.336,1.827-1.153
		c0-0.721-0.528-1.202-1.683-1.202c-0.481,0-0.817,0.048-1.01,0.097V166.728z"></path>
</g>
<g>
	<path fill="#4A2469" d="M156.248,210.816h-3.273v-1.182h7.967v1.182h-3.289v9.577h-1.404V210.816z"></path>
	<path fill="#4A2469" d="M171.209,215.668c-0.08-1.5-0.176-3.303-0.159-4.645h-0.049c-0.367,1.261-0.814,2.602-1.355,4.086
		l-1.9,5.221h-1.053l-1.741-5.124c-0.511-1.517-0.941-2.906-1.245-4.183h-0.032c-0.03,1.342-0.11,3.145-0.207,4.758l-0.288,4.612
		h-1.324l0.75-10.759h1.772l1.835,5.204c0.448,1.323,0.814,2.505,1.086,3.623h0.048c0.271-1.085,0.654-2.267,1.134-3.623
		l1.916-5.204h1.771l0.671,10.759h-1.356L171.209,215.668z"></path>
</g>
</svg>
```

Reply:
Oops, obviously i wanted to put `#452E7F` as shown in the plain icons

Reply:
Oh but that's not the section of UML here ahah

Reply:
Oh lol, my bad hahahq 😅

Reply:
The color is still wrong compared to the official logo on their website though

Reply:
> What's the reason this (`#dd4814`) color got chosen as the main color, as opposed to the other colors from the official logo (`#962444`, `#FBBD16`, `#4A2469`, `#452E7F`)? Can't find the color anywhere in their logo.
> 
> Official logo: https://www.uml.org/
> 
> ```
> <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="121px" height="88px" viewBox="0 0 396 288" enable-background="new 0 0 396 288" xml:space="preserve">
> <polygon fill="#FBBD16" points="208.096,234.496 208.096,158.167 222.729,240.603 246.553,252.477 261.867,183.949 261.867,258.923 
> 	288.753,272.151 288.753,145.616 247.914,126.278 234.982,186.663 221.937,113.84 181.323,94.391 181.323,221.04 "></polygon>
> <polygon fill="#452E7F" points="292.496,145.583 335.152,123.418 335.152,215.238 394.142,184.709 394.369,219.31 292.611,272.796 
> 	"></polygon>
> <path fill="#962444" d="M275.298,12.158l-30.163,26.669c0,0,58.851,30.25,66,33.825c7.15,3.574,17.325,10.449,6.325,20.625
> 	c-11,10.175-23.1,8.8-31.35,4.675S216.81,63.027,216.81,63.027l-31.074,26.399c0,0,59.949,30.525,69.85,35.75
> 	c9.9,5.226,46.75,18.15,81.675-9.075c34.925-27.225,29.976-37.125,29.976-39.05s-0.825-12.1-6.875-18.425
> 	c-6.051-6.325-23.65-15.4-31.075-19.25S275.298,12.158,275.298,12.158z"></path>
> <path fill="#4A2469" d="M50.154,100.832v18.721c0,2.123-0.057,3.614-0.17,4.473c-0.113,0.859-0.448,1.741-1.006,2.646
> 	c-0.557,0.906-1.29,1.594-2.202,2.06c-0.911,0.468-1.985,0.7-3.222,0.7c-1.369,0-2.577-0.275-3.626-0.829
> 	c-1.048-0.555-1.831-1.275-2.352-2.163c-0.519-0.889-0.825-1.825-0.92-2.812s-0.141-3.061-0.141-6.221v-16.575h5.968v21.002
> 	c0,1.223,0.055,2.005,0.162,2.344c0.107,0.341,0.326,0.511,0.653,0.511c0.374,0,0.615-0.188,0.725-0.563
> 	c0.107-0.375,0.161-1.26,0.161-2.655v-20.639H50.154z"></path>
> <path fill="#4A2469" d="M69.866,100.832v28.007h-5.237l-3.104-12.732v12.732h-4.98v-28.007h5.001l3.34,12.611v-12.611H69.866z"></path>
> <path fill="#4A2469" d="M82.308,100.832v28.007h-5.969v-28.007H82.308z"></path>
> <path fill="#4A2469" d="M86.875,100.832h12.017v5.612h-4.929v5.297h4.39v5.337h-4.39v11.761h-7.088V100.832z"></path>
> <path fill="#4A2469" d="M110.365,100.832v28.007h-5.969v-28.007H110.365z"></path>
> <path fill="#4A2469" d="M116.83,100.832h9.942v5.612h-3.974v5.297h3.717v5.337h-3.717v6.148h4.368v5.612H116.83V100.832z"></path>
> <path fill="#4A2469" d="M132.965,100.832h4.47c2.886,0,4.836,0.161,5.854,0.483c1.018,0.324,1.791,0.854,2.32,1.592
> 	s0.86,1.56,0.994,2.466c0.132,0.905,0.198,2.684,0.198,5.336v9.809c0,2.516-0.097,4.195-0.291,5.043
> 	c-0.194,0.849-0.533,1.512-1.015,1.99c-0.482,0.479-1.079,0.813-1.788,1.004c-0.709,0.189-1.778,0.284-3.207,0.284h-7.535V100.832z
> 	 M138.935,105.614v18.442c0.864,0,1.397-0.211,1.598-0.633c0.199-0.421,0.299-1.566,0.299-3.437v-10.892
> 	c0-1.271-0.033-2.084-0.1-2.441s-0.219-0.621-0.456-0.789C140.038,105.699,139.59,105.614,138.935,105.614z"></path>
> <path fill="#4A2469" d="M20.979,146.977v28.009h-5.217l-0.009-18.909l-2.067,18.909H9.969l-2.186-18.494l-0.008,18.494H2.557
> 	v-28.009h7.725c0.228,1.688,0.464,3.676,0.709,5.959l0.847,7.119l1.359-13.078H20.979z"></path>
> <path fill="#4A2469" d="M41.391,163.386c0,2.813-0.054,4.805-0.163,5.976c-0.108,1.169-0.449,2.239-1.021,3.207
> 	c-0.571,0.969-1.345,1.712-2.318,2.231c-0.975,0.519-2.109,0.778-3.405,0.778c-1.229,0-2.333-0.245-3.312-0.736
> 	c-0.979-0.489-1.766-1.223-2.362-2.204c-0.595-0.98-0.95-2.046-1.063-3.199c-0.113-1.154-0.17-3.171-0.17-6.053v-4.791
> 	c0-2.813,0.054-4.804,0.162-5.974c0.11-1.171,0.449-2.239,1.022-3.209c0.572-0.968,1.346-1.712,2.318-2.23
> 	c0.975-0.52,2.109-0.778,3.404-0.778c1.23,0,2.335,0.246,3.313,0.735c0.978,0.49,1.765,1.226,2.361,2.205
> 	c0.596,0.979,0.95,2.046,1.063,3.199c0.113,1.152,0.17,3.17,0.17,6.052V163.386z M35.422,154.189c0-1.302-0.06-2.135-0.178-2.497
> 	c-0.119-0.363-0.363-0.545-0.732-0.545c-0.313,0-0.552,0.147-0.719,0.44c-0.166,0.294-0.249,1.161-0.249,2.602v13.066
> 	c0,1.625,0.055,2.627,0.163,3.007c0.109,0.382,0.363,0.572,0.762,0.572c0.408,0,0.669-0.221,0.783-0.658
> 	c0.112-0.437,0.17-1.479,0.17-3.128V154.189z"></path>
> <path fill="#4A2469" d="M47.959,146.977h4.47c2.886,0,4.837,0.162,5.854,0.484c1.018,0.324,1.791,0.854,2.32,1.593
> 	c0.529,0.737,0.86,1.559,0.993,2.464c0.134,0.906,0.199,2.685,0.199,5.337v9.809c0,2.516-0.097,4.196-0.291,5.044
> 	c-0.193,0.848-0.532,1.509-1.015,1.988c-0.482,0.479-1.078,0.813-1.788,1.003c-0.709,0.19-1.778,0.287-3.207,0.287h-7.535V146.977z
> 	 M53.929,151.76v18.442c0.865,0,1.397-0.211,1.597-0.633c0.2-0.42,0.3-1.566,0.3-3.438V155.24c0-1.27-0.033-2.083-0.1-2.441
> 	s-0.218-0.62-0.457-0.788C55.031,151.844,54.585,151.76,53.929,151.76z"></path>
> <path fill="#4A2469" d="M68.369,146.977h12.072v5.613h-4.824v5.297h4.512v5.337h-4.512v6.147h5.303v5.614H68.369V146.977z"></path>
> <path fill="#4A2469" d="M92.532,146.977v22.395h4.715v5.614H84.795v-28.009H92.532z"></path>
> <path fill="#4A2469" d="M106.169,146.977v28.009h-5.97v-28.009H106.169z"></path>
> <path fill="#4A2469" d="M126.245,146.977v28.009h-5.238l-3.103-12.733v12.733h-4.981v-28.009h5.001l3.341,12.612v-12.612H126.245z"></path>
> <path fill="#4A2469" d="M146.705,157.294h-5.969v-2.538c0-1.6-0.058-2.602-0.17-3.005c-0.113-0.402-0.381-0.604-0.804-0.604
> 	c-0.366,0-0.615,0.173-0.747,0.519s-0.197,1.233-0.197,2.662v13.43c0,1.256,0.065,2.083,0.196,2.479
> 	c0.132,0.397,0.395,0.599,0.789,0.599c0.431,0,0.724-0.226,0.877-0.675c0.155-0.45,0.232-1.325,0.232-2.629v-3.319h-1.186v-4.25
> 	h6.978v15.023h-3.765l-0.555-2.007c-0.406,0.865-0.917,1.515-1.535,1.947c-0.617,0.435-1.347,0.652-2.186,0.652
> 	c-1,0-1.938-0.296-2.81-0.893c-0.872-0.593-1.535-1.326-1.987-2.203c-0.453-0.878-0.735-1.796-0.85-2.758
> 	c-0.112-0.963-0.17-2.408-0.17-4.333v-8.319c0-2.673,0.118-4.615,0.354-5.826s0.914-2.321,2.034-3.329
> 	c1.119-1.009,2.566-1.514,4.342-1.514c1.748,0,3.196,0.438,4.349,1.313c1.152,0.878,1.903,1.918,2.252,3.121
> 	c0.351,1.206,0.525,2.954,0.525,5.247V157.294z"></path>
> <path fill="#4A2469" d="M11.112,192.022v22.395h4.833v5.613H3.177v-28.008H11.112z"></path>
> <path fill="#4A2469" d="M31.58,191.705l3.424,28.009h-6.104l-0.318-5.04h-2.128l-0.349,5.04h-6.183l3.048-28.009H31.58z
> 	 M28.429,209.713c-0.297-3.168-0.6-7.089-0.907-11.767c-0.593,5.369-0.968,9.29-1.127,11.767H28.429z"></path>
> <path fill="#4A2469" d="M53.164,192.022v28.008h-5.983l-3.545-12.734v12.734h-5.69v-28.008h5.712l3.817,12.61v-12.61H53.164z"></path>
> <path fill="#4A2469" d="M73.392,202.34h-6.785v-2.54c0-1.599-0.064-2.599-0.193-3.005c-0.128-0.402-0.434-0.603-0.913-0.603
> 	c-0.417,0-0.699,0.173-0.85,0.519c-0.149,0.347-0.225,1.233-0.225,2.661v13.431c0,1.255,0.075,2.083,0.225,2.479
> 	c0.147,0.397,0.446,0.599,0.895,0.599c0.49,0,0.823-0.226,1-0.675c0.175-0.45,0.264-1.324,0.264-2.63v-3.317H65.46v-4.249h7.932
> 	v15.021h-4.279l-0.631-2.007c-0.461,0.865-1.043,1.515-1.744,1.947c-0.704,0.435-1.532,0.652-2.487,0.652
> 	c-1.136,0-2.199-0.296-3.192-0.893c-0.992-0.593-1.745-1.326-2.26-2.203s-0.837-1.796-0.965-2.757
> 	c-0.13-0.964-0.193-2.41-0.193-4.334v-8.318c0-2.674,0.133-4.616,0.401-5.827c0.269-1.21,1.04-2.32,2.313-3.33
> 	c1.272-1.009,2.917-1.512,4.937-1.512c1.986,0,3.636,0.437,4.945,1.314c1.311,0.874,2.163,1.914,2.561,3.12
> 	c0.397,1.203,0.596,2.952,0.596,5.246V202.34z"></path>
> <path fill="#4A2469" d="M92.757,192.022v18.721c0,2.122-0.063,3.614-0.186,4.472c-0.124,0.859-0.49,1.741-1.099,2.647
> 	c-0.609,0.905-1.411,1.593-2.405,2.059c-0.997,0.468-2.17,0.702-3.521,0.702c-1.496,0-2.817-0.276-3.962-0.83
> 	c-1.146-0.553-2.002-1.275-2.568-2.164c-0.569-0.888-0.904-1.823-1.008-2.812c-0.103-0.986-0.153-3.06-0.153-6.22v-16.575h6.522
> 	v21.001c0,1.223,0.058,2.006,0.178,2.344c0.116,0.342,0.354,0.513,0.713,0.513c0.409,0,0.673-0.188,0.789-0.564
> 	c0.117-0.372,0.178-1.26,0.178-2.655v-20.638H92.757z"></path>
> <path fill="#4A2469" d="M108.235,192.022l3.424,28.008h-6.104l-0.319-5.04h-2.128l-0.349,5.04h-6.182l3.048-28.008H108.235z
> 	 M105.085,210.028c-0.298-3.167-0.6-7.089-0.908-11.765c-0.593,5.368-0.969,9.29-1.126,11.765H105.085z"></path>
> <path fill="#4A2469" d="M130.512,202.34h-6.65v-2.54c0-1.599-0.063-2.599-0.188-3.005c-0.126-0.402-0.423-0.603-0.896-0.603
> 	c-0.407,0-0.686,0.173-0.832,0.519c-0.146,0.347-0.22,1.233-0.22,2.661v13.431c0,1.255,0.073,2.083,0.219,2.479
> 	c0.146,0.397,0.438,0.599,0.878,0.599c0.48,0,0.807-0.226,0.979-0.675c0.173-0.45,0.259-1.324,0.259-2.63v-3.317h-1.321v-4.249
> 	h7.773v15.021h-4.194l-0.617-2.007c-0.452,0.865-1.022,1.515-1.711,1.947c-0.688,0.435-1.5,0.652-2.435,0.652
> 	c-1.114,0-2.158-0.296-3.129-0.893c-0.974-0.593-1.712-1.326-2.216-2.203c-0.505-0.877-0.82-1.796-0.946-2.757
> 	c-0.125-0.964-0.188-2.41-0.188-4.334v-8.318c0-2.674,0.132-4.616,0.396-5.827c0.262-1.21,1.017-2.32,2.264-3.33
> 	c1.247-1.009,2.86-1.512,4.838-1.512c1.947,0,3.562,0.437,4.846,1.314c1.284,0.874,2.12,1.914,2.51,3.12
> 	c0.39,1.203,0.584,2.952,0.584,5.246V202.34z"></path>
> <path fill="#4A2469" d="M134.736,192.022h12.071v5.612h-4.823v5.299h4.512v5.336h-4.512v6.147h5.303v5.613h-12.551V192.022z"></path>
> <g>
> 	<path fill="#6D6E71" d="M393.104,167.064c0,4.471-3.509,7.979-8.076,7.979c-4.519,0-8.123-3.509-8.123-7.979
> 		c0-4.375,3.604-7.884,8.123-7.884C389.596,159.181,393.104,162.689,393.104,167.064z M378.924,167.064
> 		c0,3.509,2.596,6.298,6.153,6.298c3.461,0,6.009-2.789,6.009-6.25c0-3.51-2.548-6.346-6.058-6.346
> 		C381.52,160.767,378.924,163.603,378.924,167.064z M383.779,171.198h-1.827v-7.883c0.722-0.146,1.731-0.241,3.029-0.241
> 		c1.489,0,2.163,0.241,2.739,0.577c0.433,0.336,0.769,0.961,0.769,1.73c0,0.864-0.672,1.538-1.634,1.827v0.096
> 		c0.77,0.289,1.202,0.864,1.442,1.923c0.24,1.202,0.385,1.683,0.576,1.971h-1.97c-0.241-0.288-0.385-1.01-0.626-1.923
> 		c-0.144-0.865-0.625-1.25-1.635-1.25h-0.864V171.198z M383.827,166.728h0.865c1.01,0,1.827-0.336,1.827-1.153
> 		c0-0.721-0.528-1.202-1.683-1.202c-0.481,0-0.817,0.048-1.01,0.097V166.728z"></path>
> </g>
> <g>
> 	<path fill="#4A2469" d="M156.248,210.816h-3.273v-1.182h7.967v1.182h-3.289v9.577h-1.404V210.816z"></path>
> 	<path fill="#4A2469" d="M171.209,215.668c-0.08-1.5-0.176-3.303-0.159-4.645h-0.049c-0.367,1.261-0.814,2.602-1.355,4.086
> 		l-1.9,5.221h-1.053l-1.741-5.124c-0.511-1.517-0.941-2.906-1.245-4.183h-0.032c-0.03,1.342-0.11,3.145-0.207,4.758l-0.288,4.612
> 		h-1.324l0.75-10.759h1.772l1.835,5.204c0.448,1.323,0.814,2.505,1.086,3.623h0.048c0.271-1.085,0.654-2.267,1.134-3.623
> 		l1.916-5.204h1.771l0.671,10.759h-1.356L171.209,215.668z"></path>
> </g>
> </svg>
> ```

@Snailedlt Sorry, but `#452E7F` is the purple of the official SVG, you even wrote it 😁

Reply:
Oh yeah, lol!
You're right! Just gonna check the other colors as well, and resolve this if they are the same

Reply:
> Oh yeah, lol! You're right! Just gonna check the other colors as well, and resolve this if they are the same

No problem 😁